model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -678,12 +675,12 @@ message Schema {
   }
 
   message FieldType {
-    TypeName type_name = 1;
-    bool nullable = 2;
+    bool nullable = 1;
     oneof type_info {
-      FieldType collection_element_type = 3;
+      AtomicType atomic_type = 2;
+      ArrayType array_type = 3;
       MapType map_type = 4;
-      Schema row_schema = 5;
+      Schema row_type = 5;
```

Comment:
Proto best practice I _think_ is to go ahead and have a `RowType` message with one field. It has overhead, yes.

Reply:
Done. Agreed this is much cleaner